frappe/utils/print_format.py

```diff
@@ -19,14 +22,53 @@
 
 
 @frappe.whitelist()
-def download_multi_pdf(doctype, name, format=None, no_letterhead=False, letterhead=None, options=None):
+def download_multi_pdf(
```

Comment:
We should keep supporting previous behaviour and introduce background generation as flag. 

Otherwise it will break other code: https://sourcegraph.com/search?q=context:global+download_multi_pdf+repo:%5Egithub%5C.com/frappe/.*&patternType=standard&sm=0&groupBy=repo